tools/scripts/start-local-registry.ts

```diff
@@ -1,111 +0,0 @@
-/**
- * This script starts a local registry for e2e testing purposes.
- * It is meant to be called in jest's globalSetup.
- */
-import { execFileSync, execSync } from 'child_process';
-import { executeProcess } from '../../packages/utils/src';
-import { RegistryOptions, RegistryResult } from './types';
-import {
-  configureRegistry,
-  findLatestVersion,
-  parseRegistryData,
-  unconfigureRegistry,
-} from './utils';
-
-export default async ({
-  localRegistryTarget,
-  storage = './tmp/local-registry/storage',
-}: RegistryOptions) => {
-  const registryResult = await startLocalRegistry({
-    localRegistryTarget,
-    storage,
-    verbose: true,
-  });
-  const { stop, ...registryData } = registryResult;
-
-  console.info('Registry started:');
-  console.table(registryData);
-
-  global.stopLocalRegistry = stop;
-
-  // Publish all
-  execFileSync(
-    'npx',
-    [
-      'nx',
-      'run-many',
-      '--targets=publish',
-      `--ver=${findLatestVersion()}`,
-      '--tag=e2e',
-    ],
-    { env: process.env, stdio: 'inherit', shell: true },
-  );
-};
-
-// soft copy from https://github.com/nrwl/nx/blob/16.9.x/packages/js/src/plugins/jest/start-local-registry.ts
-// original function does not work, because it uses require.resolve('nx') and fork,
-// and it does not work with vite
-function startLocalRegistry({
```

Comment:
moved to [`tools/src/verdaccio/start-local-registry.ts`](https://github.com/code-pushup/cli/pull/798/files#diff-e6f8f291e0c903d42a54735b26540c0dec0480abcb7bc34c74062a89b85698a4)